src/components/visiumQC/SlideProcessing.tsx

```diff
@@ -19,109 +19,123 @@ import CustomReactSelect, { OptionType } from '../forms/CustomReactSelect';
 
 type SlideProcessingProps = {
   comments: CommentFieldsFragment[];
-  labware: LabwareFieldsFragment;
-  labwareResultProps: CoreLabwareResult | undefined;
+  labwares: LabwareFieldsFragment[];
+  labwaresResultsProps: CoreLabwareResult[] | undefined;
   removeLabware: (barcode: string) => void;
 };
-const SlideProcessing = ({ comments, labware, labwareResultProps, removeLabware }: SlideProcessingProps) => {
+const SlideProcessing = ({ comments, labwares, labwaresResultsProps, removeLabware }: SlideProcessingProps) => {
```

Comment:
Just a minor thing - labware denotes single or multiple labware and labwares is non-existent word (which was mentioned to me in past as a review comment  as well).

Reply:
Thank you @seenanair , I will update it accordingly